src/routes/Agreements.js

```diff
@@ -45,6 +45,10 @@ class Agreements extends React.Component {
       path: 'erm/sas/${selectedAgreementId}', // eslint-disable-line no-template-curly-in-string
       fetch: false,
     },
+    terms: {
+      type: 'okapi',
+      path: 'licenses/custprops',
```

Comment:
So we're fetching from one of the `/licenses` endpoints here, which is something we haven't done before. The `licenses` endpoints are provided by `mod-licenses`, which provides them as part of the `licenses` **Okapi interface**. 

Whenever we're depending on a backend interface in UI code, we need to express that dependency somewhere. Whenever a dependency is mandatory, we do it in our frontend module's `package.json`. If you look at `ui-licenses`'s package.json, you'll see there's an `okapiInterfaces` key in the `stripes` object that lists the dependency on licenses. 

So what we need to do is copy that key/value into the `ui-agreements`'s package.json (which already expresses its dependency on the `erm` interface) to show that we're depending on that interface as well.